README.md

```diff
@@ -830,31 +1025,44 @@ for "extra data".
 
 ###### *Value type*: text
 
-##### <a name="field-name"></a> name
+</details>
+<details>
+  <summary><h5> <a name="field-name"></a> name</h5></summary>
 
 *From OpenStreetMap*
 
-The (semi-)official name of a path, of which an Edge is a part. *Not* a
-description of the path. For example, this would be the street name for a
+The (semi-)official name of an entity. *Not* a
+description of the entity. For example, this would be the street name for a
 street path or a specially-designated name for a famous footpath.
-`name="The \[X\] trail"`, for example.
+`name="The [X] trail"`, for example.
 
 ###### *Value type*: text
 
-##### <a name="field-incline"></a> incline
+</details>
+<details>
+  <summary><h5> <a name="field-incline"></a> incline</h5></summary>
 
 *From OpenStreetMap*
 
-The estimated incline over a particular path, i.e. slope, i.e. grade,
-i.e. rise over run. If derived from OpenStreetMap data, this is the
+The estimated incline over a particular path, i.e. slope, i.e. grade,
+i.e. rise over run. If derived from OpenStreetMap data, this is the
 maximum incline over the path. If derived from DEM data, it is more
 likely to be an underestimation. Positive values indicate an uphill
 climb while negative are downhill. For example, a 45 degree downhill
-value for incline would be -1.0.
+value for incline would be -1.0. For steps, you can use "up" or "down"
+to indicate the direction of the climb.
 
-###### *Value type*: numeric
+###### *Value type*: numeric or enum
+
+###### *Enumerated values*:
+
+-   up: used when the direction of the steps points upward.
+
+-   down: Used when the direction of the steps points downward.
```

Comment:
Should we specify that `"up"` and `"down"` are relative to the direction of the edge?

Reply:
Will update the documentation!

Reply:
> For steps, you can use "up" or "down" to indicate the direction of the climb relative to the direction of the edge.

> -   `up`: when a way rises upward *in the direction* of the edge
> -   `down`: when a way rises upward *against the direction* of the edge

Reply:
Documentation updated. Thanks Cy!

Reply:
@wisamuw forgot to ask... we discussed using a separate attribute like `climb=up/down` to avoid a heterogeneous type for `incline`. The schema would then reserve `incline` for numbers to avoid the overhead and design challenges of dealing with the `up` and `down` values in downstream applications. Did we want to address this in v0.2?